src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/response.tag

```diff
@@ -49,12 +49,24 @@
       </c:forEach>
     </select>
   </div>
-  <div class="${divClassType}<c:if test="${questionWithResponses.question.questionTypeConstsum}"> width-auto</c:if>">
-    ${response.submissionFormHtml}
-    <c:if test="${response.existingResponse}">
-      <input type="hidden"
-          name="<%= Const.ParamsNames.FEEDBACK_RESPONSE_ID %>-${questionWithResponses.question.qnIndx}-${response.responseIndx}"
-          value="<c:out value="${response.responseId}"/>">
-    </c:if>
-  </div>
+  <c:choose>
+    <c:when test="${questionWithResponses.question.questionTypeConstsum}">
+      ${response.submissionFormHtml}
+      <c:if test="${response.existingResponse}">
```

Comment:
The common fragment can be extracted out.

Reply:
Hello, I am not sure how to do extraction in a jsp file. I googled for a while, and realised that many websites suggest creating a jsp fragment/another tag file to do so, which may be an overkill for this instance. What do you think? Any better ideas? If not I will go ahead and create a jsp fragment...